lld/test/ELF/relro-data-unlikely.s

```diff
@@ -0,0 +1,50 @@
+# REQUIRES: x86
+# RUN: llvm-mc -filetype=obj -triple=x86_64-pc-linux %s -o %t.o
+
+# RUN: echo "SECTIONS { \
```

Comment:
There are quite a few extra files. We prefer split-file for such tests. Many tests use a.o b.o `out` as relocatable and executable file names. Consider `-o a.o`  `ld.lld -z keep-data-section-prefix -T a.lds a.o -o out`